core/arch/arm/include/mm/core_mmu.h

```diff
@@ -74,6 +77,11 @@
 #define CFG_TEE_RAM_VA_SIZE		CORE_MMU_PGDIR_SIZE
 #endif
 
+#ifdef CFG_WITH_LPAE
+#define CORE_MMU_L1_TBL_OFFSET		(CFG_TEE_CORE_NB_CORE * 4 * 8)
```

Comment:
` * 4 * 8` looks like magic, especially compared to how `NUM_L1_ENTRIES` is defined in **core_mmu_lpae.c**. Deserves a bit of explanation: why 4 64bit wide mmu descriptors ?

Regarding the armv7, the comment is straightforward: we need a full L1 table.

Reply:
Please make these comments in https://github.com/OP-TEE/optee_os/pull/2048 instead

Reply:
ok, sorry.